CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # CHANGELOG
 
+## [4.3.01](https://github.com/kokkos/kokkos/tree/4.3.01)
+[Full Changelog](https://github.com/kokkos/kokkos/compare/4.3.00...4.3.01)
+
+### Backend and Architecture Enhancements:
+
+#### HIP:
+* MI300 support unified memory support [\#6877](https://github.com/kokkos/kokkos/pull/6877)
```

Comment:
```suggestion
* MI300 support unified memory [\#6877](https://github.com/kokkos/kokkos/pull/6877)
```

Reply:
This is OK, we don't need to fix it here, the more important place to fix is develop.

Reply:
It's one less line to fix in merge conflicts for the next release.

Reply:
Yeah, to avoid a round of retesting (and possible unrelated issues popping up like system problems, connectivity etc.) I'd prefer to keep it as is